vmfiles/nuxeo.init

```diff
@@ -11,11 +11,7 @@
 
 DESC="Nuxeo"
 
-NUXEO_USER=nuxeo
-NUXEOCTL="/var/lib/nuxeo/server/bin/nuxeoctl"
-NUXEO_CONF="/etc/nuxeo/nuxeo.conf"
-export NUXEO_CONF
-
+. /etc/profile.d/nuxeo_env.sh
```

Comment:
Interesting. Don't we need those variables in non interactive Shells too? Isn't there an impact regarding this aspect?

Reply:
After reviewing the NXBT tickets, this fixes: https://jira.nuxeo.com/browse/NXBT-2127

/etc/profile.d is a standard Unix convention to provide a place for system-wide environment variables to be set, across interactive and non-interactive shells.  The lack of consistent NUXEO_CONF environment as root and nuxeo users is a source of great annoyance.  This change fixes that issue by moving common Nuxeo variables to a location used by all scripts.

Reply:
The variables set in nuxeo.init are limited to the `service nuxeo start/stop` function and are not available system-side.